helm-charts/templates/data-plane/config-deployer/config-deploy-api-scope1.yaml

```diff
@@ -0,0 +1,29 @@
+
+# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+{{- if and .Values.wso2.apk.dp.enabled .Values.wso2.apk.dp.configdeployer.enabled }}
+apiVersion: dp.wso2.com/v1alpha1
+kind: Scope
+metadata:
+  labels:
+    managed-by: "apk"
+  name: {{ template "apk-helm.resource.prefix" . }}-config-ds-scope-1
```

Comment:
shall we change name to api-create-scope

Reply:
Changes the scope name